storage/engine/engine.go

```diff
@@ -39,7 +39,7 @@ type Iterator interface {
 	// is <= the provided key.
 	SeekReverse(key MVCCKey)
 	// Valid returns true if the iterator is currently valid. An
-	// iterator which hasn't been seeked or has gone past the end of the
+	// iterator which hasn't been sought or has gone past the end of the
```

Comment:
I was pretty sure the correct thing was to stick to normal english rules for this type of thing.

`seek()ed` isn't any better.  

Instead I'll make the comment not verbify it.
`An iterator that hasn't had Seek called on it or has gone past the end of the key range is invalid.`